### ISSUE TITLE
AR-Diffusion data.name and exp.name

### ISSUE THREAD

**Issue by AIgods** [microsoft/ProphetNet#75]
in AR Difusion, I would like to ask why in train.sh, in the training script for the commongen dataset, data.name is not commongen, but iwslt14_tok, and What does exp.name mean

# Commongen
FILE_NAME = commongen
STEP = 40000

torchrun --nproc_per_node=8 --nnodes=1 ./train_utils/trainer_main.py \
model.name='bert-base-uncased' batch_size=384 grad_accum=1 \
total_steps=$STEP exp.name=$FILE_NAME \
data.name=iwslt14_tok tgt_len=54 max_pos_len=128 lr=3e-4 lr_step=40000 \
intermediate_size=512 num_attention_heads=8 dropout=0.2 \
in_channels=64 out_channels=64 time_channels=64 \
eval_interval=3000 log_interval=1000 \
schedule_sampler='xy_uniform' time_att=True att_strategy='txl' \

**Comment by hwaseem04** [comment]
Afaik, exp.name is something that you give as name for the experiment you are doing. Ideally it can be any arbitary string value.

**Comment by wutong4012** [comment]
Sorry, there is a mistake here, data.name should really be replaced with commongen.
